sdks/go/pkg/beam/core/runtime/exec/datasource.go

```diff
@@ -47,20 +47,21 @@ type DataSource struct {
 	splitIdx  int64
 	start     time.Time
 
-	// rt is non-nil if this DataSource feeds directly to a splittable unit,
-	// and receives the current restriction tracker being processed.
-	rt chan sdf.RTracker
+	// su is non-nil if this DataSource feeds directly to a splittable unit,
+	// and receives that splittable unit when it is available for splitting.
+	su chan SplittableUnit
```

Comment:
Improved this documentation for now, but this is part of concurrency code I might be refactoring to avoid all the blocking, so it might just be gone with the next commit I send.